Content.Server/GameObjects/Components/Disposal/DisposalTaggerComponent.cs

```diff
@@ -143,5 +150,37 @@ public override void OnRemove()
             base.OnRemove();
             UserInterface?.CloseAll();
         }
+
+        [Verb]
+        public sealed class ConfigureVerb : Verb<DisposalTaggerComponent>
```

Comment:
On the tagger it seems to have the same functionality as clicking it as I can't use the verb out of range.

Reply:
The verb is mainly for opening the interface as an admin ghost.